massdash/plotting/InteractivePlotter.py

```diff
@@ -8,7 +8,8 @@
 
 # Data modules
 import numpy as np
-from scipy.signal import savgol_filter, gaussian, convolve
+from scipy.signal import savgol_filter, convolve
+from scipy.signal.windows import gaussian
```

Comment:
Can you add the specific version dependency for scipy in requirements, setup.cfg and project.toml?